README.md

```diff
@@ -95,6 +96,18 @@ For a full command reference, see the list below, or visit [cli.netlify.com](htt
 
 Run any Netlify API method
 
+### [blobs](/docs/commands/blobs.md)
```

Comment:
[boulder] -  all the `docs/commands/blobs.md` links in the README give 404s

Reply:
No idea why that is. Perhaps @sarahetter can help?

Reply:
Hmm yeah we've seen this before - https://linear.app/netlify/issue/CT-297/fix-generated-docs-for-cli-recipes-command 
Thought it was just an issue with the `recipes` command but it looks like it's wider spread. I'll bump up this issue in our priority list

Reply:
I believe this must have to do with the docs generation issue I mentioned in response to the other comment

Reply:
https://github.com/netlify/cli/pull/6171 resolves this - it removes the entire README, and relies on just the cli.netlify.com docs for a command reference